internal/proxyservice/tcp/pg/handler.go

```diff
@@ -0,0 +1,80 @@
+package pg
+
+import (
+	"net"
+
+	"github.com/cyberark/secretless-broker/internal/proxyservice/tcp/pg/protocol"
+	"github.com/cyberark/secretless-broker/pkg/secretless/log"
+	"github.com/cyberark/secretless-broker/pkg/secretless/plugin/connector"
+)
+
+// clientOptions stores the option that were specified by the connection client.
+// The User and Database are required. Other options are stored in a map.
+type ClientOptions struct {
+	// TODO: remove this when custom authorization is removed.
```

Comment:
This is probably existing code, but I didn't understand the TODOs.  Can you clarify if you do?

Reply:
ClientOptions isn't actually needed. Got rid of it